### PR TITLE
Add MangaFire connector

### LINKED ISSUE #5585: [Site Request] MangaFire
**Name of the website**
thewebsitename MangaFire

**Website urls (examples below)**
- Site: https://mangafire.to/
- Manga List: https://mangafire.to/az-list
- Manga example: https://mangafire.to/manga/hitogatana.zlr2
- Chapter online viewer example: https://mangafire.to/read/sidonia-no-kishi.lmm3/en/chapter-1

**Languages**
List of languages supported by the website
English, French, Spanish, Spanish(LATAM), Portuguese, Portuguese(Br), Japanese

**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

### PR DESCRIPTION
**Pull request by Druzai**
Fixes #5585

Some pages need to be unscrambled.
Similar to `MangaReaderTo`, but for now they use far more simpler protection.

### CODE REVIEW

**Review by MikeZeDev — APPROVED** [review]
Nothing wrong for me but its a bit complex maybe ronny should have a look before merging.

File: src/web/mjs/connectors/MangaFire.mjs
Comment (27136773) [inline src/web/mjs/connectors/MangaFire.mjs]: The `element.pathname` property might be a better choice as it provides only the least necessary information

File: src/web/mjs/connectors/MangaFire.mjs
Comment (27136773) [inline src/web/mjs/connectors/MangaFire.mjs]: Can be simplified to `canvas.toBlob(resolve, type, quality)`

**Review by 27136773 — APPROVED** [review]

**Comment by JumpIn-Git** [comment]
the connector isnt showing?

**Comment by MikeZeDev** [comment]
Merged PR are not here to report bugs. Open an issue for that.

**Comment by 27136773** [comment]
[Missing in HaruNeko](https://github.com/search?q=repo%3Amanga-download%2Fharuneko%20%27mangafire%27&type=code)

**Pull request merged**
